.github/ISSUE_TEMPLATE/release.md

```diff
@@ -1,59 +1,59 @@
 ---
-name: k6 Release 
-about: k6 release accommodates activities and a checklist that has the k6 open-source release process.
+name: k6 Release
+about: k6 release accommodates activities and a checklist with the k6 open-source release process.
 title: 'k6 release v0.4x.0'
 ---
 
-**Release Date**: 
+**Release Date**:
 
 25th May 2023 <<  TODO: WRITE HERE THE UPDATED RELEASE DATE
 
 ## Release Activities
 
 ### At the beginning of the cycle
 
-- [ ] Create a new branch `release-notes-v04x` and add a new related [release notes file](release-template/release%20notes) using the available [template](release-template/release%20notes/template.md).
-- [ ] Go through the potential [dependencies updates](Dependencies.md) then create a PR if any is relevant. 
+- [ ] Create a new `release-notes-v04x` branch and add a new [release notes file](release-template/release%20notes) using the available [template](release-template/release%20notes/template.md) to the [repository's `release notes` folder](https://github.com/grafana/k6/tree/cbd9e9ad184f19fdd84c7041f8084218ce4b738f/release%20notes).
+- [ ] Go through the potential [dependencies updates](Dependencies.md) and create a dedicated PR if any of them is relevant to this release.
 
 ### Release Preparation
 
-~ 1 week before the release date.
+#### ~ 1 week before the release date.
 
-- [ ] k6-docs PRs for all new or changed functionality have been created.
-- [ ] All PRs to the k6 repository in the current [milestone](https://github.com/grafana/k6/milestones) have been merged.
-- [ ] A pull request with the release notes and request the final reviews (including the @k6-browser, devrels folks).
-- [ ] Open a separate PR for bumping [the version](https://github.com/grafana/k6/blob/9fa50b2d1f259cdccff5cc7bc18a236d31c345ac/lib/consts/consts.go#L11).
-- [ ] The release notes PR shared in the `#k6-oss-dev` internal channel mentioning all the impacted teams (@k6-browser, @k6-chaos, @k6 devrel and any other potential stackholder of the new release).
-- [ ] `DefinitelyTyped/DefinitelyTyped` PR(s) is ready.
+- [ ] Ensure all PRs in the k6-docs repository, related to new or modified functionalities introduced by the new version have been merged.
```

Comment:
Merging them this early  means:
1. we can't update docs in that time
2. we will release changes for the version before we release

So `merging` should likely wait until the day of the release.

Coordinating around "let's not do any updates to the docs for the next week" will be fairly hard and blocking a lot of people. 

We can have a branch with all the changes in one place that we merge at the end ... But  I feel that will be even harder to fix merge conflicts, so not really better IMO

Reply:
Good catch, I didn't realize I had used "merged" instead of "created". I'm gonna fix it indeed 👍🏻

Reply:
I've replaced "merged" with "created".